### ISSUE TITLE
[SUGESTÃO] Utilizar um CustomIcon como Badge para a linguagem Java em "badge.md"

### ISSUE THREAD

**Issue by WillCoutinho** [elidianaandrade/dio-lab-open-source#5220]
**Sua sugestão está relacionada a um problema? Por favor descreva.**
O ícone oficial do Java foi removido do site [Simple Icon](https://simpleicons.org) (que é utilizado pelo [Shields.io](https://shields.io/docs/logos#simpleicons)) por questões legais.
Os detalhes da remoção estão nesse issue: https://github.com/simple-icons/simple-icons/issues/7374
Com isso, o arquivo `badge.md` dentro da pasta `utils.badge` está sem ícone para a linguagem Java.

**Descreva a solução que você gostaria**
Como workaround, é possível utilizar o logo do OpenJDK ou gerar um logo em base64 e utilizar diretamente como um [Custom Logo](https://shields.io/docs/logos#custom-logos).

**Descreva as alternativas que você considerou**
Eu fiz um Custom Logo do Java caso tenham interesse: [Badge Java](https://img.shields.io/badge/Java-orange.svg?logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAOxAAADsQBlSsOGwAAAxxJREFUWIXF112IlGUUB/Dfrlumsju2gboRRRsZGJGkJGhYF0VEFBaBIAVFYRdB3ZVIdCOoXUiI6IUIgnhjhVIUEllREEUQUtGHBZLaF+ZX6brmTjtdnDPOu9OOOzPs6B9e3neeeZ/zP895zvmf52XyMLedSd2T6MAzk2irZQzgNLouB3kP3sOJy0E+FW+igu2XmnwOPk3yv3BD4b8S7sL0TpHfiV+SvIxlmIEX8AW+w6JOkT+Ms0n+L57Gffg1x3bo4MqX4p8C+UoswTmM4pVOEVfxfZJX8FKOVfNgR6fJSwXyD3LsCpEDFTzVjtFWlPBv/J7Pu/Jexpl8frQdB6a0+H63SLifsTfHqiV3Mz7E4XYcaRY9+ETUfSnHpqnlxu5OklfRj/14uTD2UDrwWavG2umGJ3AP7sCsHDuY99fbsNcQs/Cc0Ph9WJ2kVad7cFs+v4a1xnbDbqzA1/gWT4xH0qh9zhH7OjN/v4EtYvVnheh0iWbUjxsxlPOuxQg2iAoZwMcYFIeWarQuiquwUyhcpcmrjJ+wUfSFItbnO8uajUAV00WYF6b3R0RoR0QkTuGoaEyHcL6Bnfdxr9jC/RNw/g8rRHi3qSVdEV24Ho+IY1lP3f/3iy37qFXiKtalgUre/8SP+EFEpH6b3hFbCPNF3uzDNeMZb/YMNw8P4Hax2pJY6TkcF8r4jWhMX6UjC0WrfltNNdvGoFDArWqrk04M4jFsEns8KRjA3UJwevGqsdl+XCRg2djwj4hcqEfTp+XZ4oQ7miQbxOpKeFHs5WFR38M4JkRmD1bh1gZ23xWaMSHWqK1mTbNeXwRXiuiVRVQnxC34TS3j9wo5XoSrm5g/VSTscmwW54chPFt4p684Yby9KeWE5aKMig3rjAj7aVEBRCL2poP9aXMEX4oK2I4/0rlVQuIvNK2JkqMkSm8urhO1PFOEdkoSDeOkmiIeEHkxnDZm43E8j7fyfgH1qlWPISE8fbnygyIJTzV4v1d8oDyIBaKSFggVfNI4atjKx+QAFguBuUlEo09EaUZeoyISR0QUPhfHtKONjP4HxJ7K18iRRpEAAAAASUVORK5CYII=). 
Para alterar a cor padrão basta trocar no início da URI a cor `orange` para outra de preferência, mas a cor do ícone customizado é preto e não é possível alterá-lo (somente gerando um novo base64 usando o ícone de outra cor). 

**Contexto adicional**
N/A

**Comment by mondeja** [comment]
You can use the OpenJDK logo in a badge changing the color using https://shields.io, for example:

```
![Static Badge](https://img.shields.io/badge/any_text-you_like-blue?logo=openjdk&logoColor=lightgreen)
```

![Static Badge](https://img.shields.io/badge/any_text-you_like-blue?logo=openjdk&logoColor=lightgreen)

**Comment by WillCoutinho** [comment]
> You can use the OpenJDK logo in a badge changing the color using https://shields.io, for example:
> 
> ```
> ![Static Badge](https://img.shields.io/badge/any_text-you_like-blue?logo=openjdk&logoColor=lightgreen)
> ```
> 
> ![Static Badge](https://camo.githubusercontent.com/ce7883cc7296bb5b6209956814841cbd11f45b4838eafbfcf4058c94a2d2cfc8/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f616e795f746578742d796f755f6c696b652d626c75653f6c6f676f3d6f70656e6a646b266c6f676f436f6c6f723d6c69676874677265656e)

Yep, and this solution (using openJDK logo) was mentioned here: https://github.com/elidianaandrade/dio-lab-open-source/issues/5070. 
But for those who like/want the "old" Java icon, this suggestion fits well. Thanks for your input! 😄
